core/test/test_manifoldmath.cpp

```diff
@@ -3,70 +3,74 @@
 #include <engine/Vectormath.hpp>
 #include <engine/Vectormath_Defines.hpp>
 
+using Catch::Matchers::WithinAbs;
+
 TEST_CASE( "Manifold operations", "[manifold]" )
 {
+    Catch::StringMaker<float>::precision  = 12;
+    Catch::StringMaker<double>::precision = 12;
+
     int N       = 10000;
     int N_check = std::min( 100, N );
     vectorfield v1( N, Vector3{ 0.0, 0.0, 1.0 } );
     vectorfield v2( N, Vector3{ 1.0, 1.0, 1.0 } );
 
-    REQUIRE( Engine::Vectormath::dot( v1, v2 ) == Approx( N ) );
+    REQUIRE_THAT( Engine::Vectormath::dot( v1, v2 ), WithinAbs( N, 1e-12 ) );
```

Comment:
Need to set the `1e-12` epsilon value dependent on the precision, otherwise tests will fail when using single precision (`float`)